apps/chat/src/components/Common/ApplicationWizard/CodeAppView/CodeAppView.tsx

```diff
@@ -183,6 +181,41 @@ export const CodeAppView: FC<ViewProps> = ({
     ],
   );
 
+  const handleSubmit = useCallback(
+    (data: FormData) => {
+      const preparedData = getApplicationData(data, type);
+
+      if (type === ApplicationType.CODE_APP) {
+        preparedData.functionStatus = selectedApplication?.functionStatus;
+      }
+
+      if (isEdit) {
+        if (
+          selectedApplication?.isShared &&
+          type === ApplicationType.CODE_APP &&
+          preparedData.function?.sourceFolder !==
+            selectedApplication.function?.sourceFolder &&
+          selectedApplication
```

Comment:
already cheched here `selectedApplication?.isShared`
```suggestion
          selectedApplication?.isShared &&
          type === ApplicationType.CODE_APP &&
          preparedData.function?.sourceFolder !==
            selectedApplication.function?.sourceFolder
```